packages/flutter/lib/src/widgets/routes.dart

```diff
@@ -2209,6 +2245,12 @@ class _RenderFocusTrap extends RenderProxyBoxWithHitTestBehavior {
     if (focusNode == null || result == null)
       return;
 
+    if (event.kind == PointerDeviceKind.touch && focusNode is FocusScopeNode)
+      // If a controller is initially provided then the parent stops doing so for rebuilds,
+      // a new instance of CupertinoTabController should be created and used by the widget,
+      // while preserving the previous controller's tab index.
```

Comment:
Can you explain this to me?  I'm not sure exactly how this comment relates to the `if` here.

Reply:
Thanks for pointing out this negligence. Would you be so kind as to check the fixed comment is appropriate?

When a test case 'Drawer state query correctly' in `scaffold_test` runs, it tries to get renderObject of inactive element while dispatching a pointer event and throws an exception.

The `if` here protects `findRenderObject` from calling in an inactive state.

[Linux web_tests_3](https://logs.chromium.org/logs/flutter/buildbucket/cr-buildbucket/8818264333803275777/+/u/run_test.dart_for_web_tests_shard_and_subshard_3/test_stdout) describes the exception details.